platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/CaptureJniLibrary.kt

```diff
@@ -229,6 +229,19 @@ internal object CaptureJniLibrary {
         durationS: Double,
     )
 
+    /**
+     * Writes an app launch TTI log. The method should be called only once per logger Id. Consecutive calls
+     * have no effect.
+     *
+     * @param loggerId the ID of the logger to write to.
+     * @param durationS the time between a user's intent to launch the app and when the app becomes
+     *                  interactive.
+     */
+    external fun writeAppLaunchTTILog(
+        loggerId: Long,
+        durationS: Double,
```

Comment:
why the `S` in the param name? Is it in Seconds?

Reply:
I see below that it is. Is this a standard, to use seconds in Doubles to track time? and if so we should add the seconds unit part to the docstring

Reply:
also worth mentioning that the value should be positive otherwise will be ignored

Reply:
Anything that an API allows users will do. My opinion is that we should never allow for invalid operations using the type system itself e.g. our Span API doesn't allow negative durations. 

So at least we should document that this particular usage is invalid